client/whispers/whisper-reducer.ts

```diff
@@ -104,6 +104,10 @@ export default immerKeyedReducer(DEFAULT_STATE, {
       text,
     })
 
+    // Reorder the sessions to put the one that got the message on top of the list
+    state.sessions.delete(target)
```

Comment:
Unnecessary, by putting it first below it'll keep it first in the order and since it's a set it won't have a duplicate